src/cr/CredentialRequest.js

```diff
@@ -31,11 +31,11 @@ class CredentialRequest {
     return new CredentialRequest(null, null, R.clone(obj));
   }
 
-  acceptClaims(claims = []) {
-    const claimInstances = claims.map((claim) => {
+  async acceptClaims(claims = []) {
+    const claimInstances = await Promise.all(claims.map(async (claim) => {
```

Comment:
await Promise.all(claims.map(async (claim)

this is really hard to read

Reply:
I've split it onto a seperate line